index.html

```diff
@@ -58,6 +58,7 @@ <h2>Por que iniciar uma parceria com a Data Bureau?</h2>
                 <li>Desenvolvemos soluções personalizadas para projetos inéditos;</li>
                 <li>Seguimos prazos e entregas pontuais previamente agendadas;</li>
                 <li>Temos preços acessíveis para se adequar ao seu orçamento.</li>
+                <li>Descontos especiais para pesquisas de IC, Mestrado e Doutorado.</li>
```

Comment:
Eu posso, por exemplo, comentar essa linha e te perguntar motivos de ter a feito ou sugestões de como a fazer e você pode responder logo abaixo, assim os devs debatem sem ter que ficar indo e voltando código: tudo se debate aqui e o dono do PR vai ajeitando na máquina dele e subindo aqui o que for parecendo necessário mudar

Reply:
ah, entendi. Boa né? bem organizado